deepspeed/profiling/flops_profiler/profiler.py

```diff
@@ -478,40 +478,47 @@ def _prod(dims):
     return p
 
 
+def torch_numel(tensor):
+    n = torch.numel(tensor)
+    if isinstance(n, torch.Tensor):
+        n = n.item()
+    return n
+
+
 def _linear_flops_compute(input, weight, bias=None):
     out_features = weight.shape[0]
-    macs = torch.numel(input) * out_features
+    macs = torch_numel(input) * out_features
```

Comment:
Would `input.numel()` not work?

Reply:
It should? I wasn't sure the reasoning behind using `torch.numel()` in this case so opted to preserve the original style. @cli99 might know more here?

Reply:
input.numel() works as well

Reply:
Cool! I’ll switch the PR to that style then